### PR TITLE
Summer '22 release

### PR DESCRIPTION
**Pull request by satyasekharcvb**
### What does this PR do?
- API version updates
- fixed UI issue in eventBubbling component
- Added new recipes for light DOM
- Added recipes for Notification Modules
 
## The PR fulfills these requirements:

[ X ] Tests for the proposed changes have been added/updated.
[ X ] Code linting and formatting were performed.

### CODE REVIEW

File: force-app/test/jest-mocks/lightning/alert.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
```
Comment (pozil) [inline force-app/test/jest-mocks/lightning/alert.js:6]: Drop copyright header on the 4 mock files.

File: force-app/main/default/lwc/lightDomHello/lightDomHello.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<template lwc:render-mode="light">
+    <lightning-card title="LightDomHello" icon-name="custom:custom5">
+        <div class="slds-var-m-around_medium">Hello from Light DOM!</div>
```
Comment (pozil) [inline force-app/main/default/lwc/lightDomHello/lightDomHello.html:3]: Can you sync with @adityanaag3 and align the recipes with LWC Recipes OSS?

**Review by pozil — CHANGES_REQUESTED** [review]
Thanks for the update @satyasekharcvb and good call on renaming the toast notification recipe. I left two change requests.

File: force-app/test/jest-mocks/lightning/alert.js (hunk 1 above)
Comment (satyasekharcvb) [reply force-app/test/jest-mocks/lightning/alert.js]: Changes Done!

File: force-app/main/default/lwc/lightDomHello/lightDomHello.html (hunk 2 above)
Comment (satyasekharcvb) [reply force-app/main/default/lwc/lightDomHello/lightDomHello.html]: Hi @pozil, Changes Done! Thank you!

File: force-app/main/default/lwc/miscNotificationModules/__tests__/miscNotificationModules.test.js [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+            'lightning-button.alertButton'
+        );
+        alertButton.click();
+
+        // Wait for any asynchronous DOM updates
+        await flushPromises();
+
+        // Check if event has been fired
+        expect(LightningAlert.open).toHaveBeenCalled();
+        expect(LightningAlert.open.mock.calls).toHaveLength(1);
```
Comment (pozil) [inline force-app/main/default/lwc/miscNotificationModules/__tests__/miscNotificationModules.test.js:42]: I don't think that we need to check the number of time the method has been called but if you do, you can replace the two lines by just this:
```js
expect(LightningAlert.open).toHaveBeenCalledTimes(1);
```

**Review by pozil — APPROVED** [review]
Thanks for the update. I'll merge this next Monday after the release.

**Pull request merged**